bindings/matrix-sdk-ffi/src/api.udl

```diff
@@ -289,9 +289,7 @@ callback interface TimelineListener {
     void on_update(TimelineDiff update);
 };
 
-interface TimelineDiff {
-    MoveData? move();
-};
```

Comment:
Was is used by ElementX before?

Reply:
I'm sure they implemented moving items, otherwise they wouldn't have been able to process all the diffs.